api/src/main/java/com/epam/pipeline/manager/cluster/ClusterApiService.java

```diff
@@ -74,6 +76,12 @@ public NodeInstance terminateNode(String name) {
         return usageMonitoringManager.getStatsForNode(nodeName, from, to);
     }
 
+    @PreAuthorize("hasRole('ADMIN') OR @grantPermissionManager.nodePermission(#nodeName, 'READ')")
```

Comment:
Name in the `PreAuthorize` section must match argument name: `#nodeName` -> `#name`

Reply:
Done